src/main/java/King.java

```diff
@@ -0,0 +1,173 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class King {
+    public static final String name = "King";
+    public static int repeatCount = 0;
+    public static int tasksCount = 0;
+    public static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void greet() {
```

Comment:
Is greet a misleading name for what this function actually does (both greet and call chat())?